app/src/main/java/org/mozilla/fenix/search/toolbar/IncreaseDpsAction.kt

```diff
@@ -0,0 +1,20 @@
+package org.mozilla.fenix.search.toolbar
```

Comment:
Please add the Mozilla license at the top of the file
```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Reply:
updated